core_backend/app/tags/models.py

```diff
@@ -18,7 +18,7 @@
 from .schemas import TagCreate
 
 content_tags_table = Table(
-    "content_tags",
+    "content-tag",
```

Comment:
Hey @lickem22, this breaks other relationships - just tried to deploy and getting errors when trying to save content

Reply:
Instead of rewriting migration script we need a new migration that changes the table name from content_tag to content-tag!

Reply:
It's not supposed to be an issue because the model still reflects the revision. What erroe do you get? I doubt it is because of the migration script

Reply:
When I add tags using UI I get this error on the backend:

> sys:1: SAWarning: Object of type <ContentDB> not in session, add operation along 'TagDB.contents' won't proceed (This warning originated from the Session 'autoflush' process, which was invoked automatically in response to a user-initiated operation.)

You are right that I didn't get any errors with the migration step when backend container started.. but why would we change the migration script? I'm not so experienced with DB migrations so just trying to understand. Isn't [core_backend/migrations/versions/2024_05_17_b5ad153a53dc_add_tags.py](https://github.com/IDinsight/aaq-core/pull/221/files#diff-bb14214a7eeaf41c3967d124ed02ddf8bf4d4284341c5043917851b67466e4ce) tied to the latest commit hash on main, and if we want to make any changes to the DB shouldn't we add another migration script?

Reply:
The goal of the scripts is to have versioning for the database schema just like git. So that when things break, you can revert easily.  Usually when you made a mistake in one of the scripts, you would delete the old migration file and recreate a new migration file. But I decided to update the migration script manually because the result would have been the same whether I generate it or update it manually. 
Also, the error you are getting, I am not able to replicate it

Reply:
I did some research and basically the reason we have the warning is because the ContentDB model was not added to the session yet. Recreating this warning is a bit tricky but I think I fixed it. Let me know if you still experience it.